contributing/getting-started-quick.md

```diff
@@ -137,7 +137,7 @@ git checkout -b <branch name> master
 5. If your reviewer requests changes make them locally and then repeat the steps in this section to push the changes to your branch back up to GitHub again.
 6. For pushes after the first, just use `git push`
 7. If you don't get a new review within 2 business days, feel free to ping the pull request by adding a comment.
-8. If you see visual diffs reported by [Percy](http://percy.io/ampproject/amphtml), and want to access the results, fill out this [form](https://docs.google.com/forms/d/e/1FAIpQLScZma6qVJtYUTqSm4KtiF3Zc-n5ukNe2GXNFqnaHxospsz0sQ/viewform).
+8. If you see visual diffs reported by [Percy](http://percy.io/ampproject/amphtml), click through the check on GitHub to access the results. Differences can be approved by a core contributor.
```

Comment:
nit: Make this more specific. `Diffs that are expected can be approved by a core contributor`